app/src/main/java/com/owncloud/android/ui/activity/FileDisplayActivity.java

```diff
@@ -2352,7 +2352,7 @@ private void openDeepLink(Uri uri) {
         DeepLinkHandler.Match match = linkHandler.parseDeepLink(uri);
         if (match == null) {
             dismissLoadingDialog();
-            DisplayUtils.showSnackMessage(this, getString(R.string.invalid_url));
```

Comment:
This way, we do not have the error message any longer.
I suggest to put it in "default" of switch (path).